tock/hours/models.py

```diff
@@ -400,6 +400,21 @@ def save(self, *args, **kwargs):
         super(TimecardObject, self).save(*args, **kwargs)
```

Comment:
I still think we should remove L376 from the save() and `submitted` from the model while we're doing this. They're not doing any good, right?